demos/hr_agent/docker-compose.yaml

```diff
@@ -0,0 +1,25 @@
+services:
+  api_server:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    environment:
+      - SLACK_BOT_TOKEN=${SLACK_BOT_TOKEN:?error}
```

Comment:
This should be optional feature and we shouldn't fail. If token is empty then just print the rest api on console.